test/_test-utils.js

```diff
@@ -20,11 +20,11 @@ export function shouldReject(f, expectedErrorMessage, testErrorMessage) {
 		console.log('got result:', inspect(res, false, null, true));
 		fail('should rejects');
 	});
-	it('instance of Error', function () {
+	it('instance of Error', () => {
```

Comment:
Here and above back to `function` instead of anonymous function, cause the last one has no context, so `this` is undefined.